lifetotals.html

```diff
@@ -0,0 +1,38 @@
+<html>
+<head>
+    <title>life totals</title>
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" type="text/css" href="lifetotals.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Roboto+Condensed" rel="stylesheet">
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"
+            integrity="sha256-cCueBR6CsyA4/9szpPfrX3s49M9vUU5BgtiJj06wt/s=" crossorigin="anonymous"></script>
+
+    <script src="https://www.gstatic.com/firebasejs/3.6.4/firebase.js"></script>
+    <script type="text/javascript" src="config.js"></script>
+    <script type="text/javascript" src="firebase.js"></script>
+    <script type="text/javascript" src="lifetotals.js"></script>
+</head>
+<body>
+<div class="player" id="p1">
+    <div class="name" id="player1_name"></div>
+    <div class="button plus plus1">+1</div>
```

Comment:
is this the order they wind up in on the page?

it should look like this:
-3 -2 -1  LIFE  +1 +2 +3